deploy/crds/enterprise.splunk.com_clustermasters_crd.yaml

```diff
@@ -2790,15 +2818,15 @@ spec:
                                     phase:
                                       description: Phase type
                                       type: string
-                                    retryCount:
-                                      description: RetryCount defines the number of
-                                        retries completed so far
-                                      format: int32
-                                      type: integer
                                     status:
                                       description: Status of the phase
                                       format: int32
                                       type: integer
+                                    tryCount:
+                                      description: RetryCount defines the number of
```

Comment:
description does not match the field name

Reply:
I missed addressing the description part, will address it.